src/applications/personalization/dashboard-2/components/claims-and-appeals/Appeal.jsx

```diff
@@ -65,43 +65,41 @@ const Appeal = ({ appeal, name }) => {
   appealTitle = capitalizeFirstLetter(appealTitle);
 
   return (
-    <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--8 medium-screen:vads-u-padding-right--3">
-      <div className="vads-u-padding-y--2p5 vads-u-padding-x--2p5 vads-u-background-color--gray-lightest">
-        <h3 className="vads-u-margin-top--0">
-          {appealTitle}
-          {/* Claim for compensation received June 7, 1999 */}
-        </h3>
-        <div className="vads-u-display--flex">
-          <i
-            aria-hidden="true"
-            className={`fas fa-fw fa-check-circle vads-u-margin-right--1 vads-u-margin-top--0p5 vads-u-color--green`}
-          />
-          <div>
+    <div className="vads-u-padding-y--2p5 vads-u-padding-x--2p5 vads-u-background-color--gray-lightest">
+      <h3 className="vads-u-margin-top--0">
+        {appealTitle}
+        {/* Claim for compensation received June 7, 1999 */}
+      </h3>
+      <div className="vads-u-display--flex">
+        <i
+          aria-hidden="true"
+          className={`fas fa-fw fa-check-circle vads-u-margin-right--1 vads-u-margin-top--0p5 vads-u-color--green`}
+        />
+        <div>
+          <p className="vads-u-margin-y--0">
+            Status: {getStatusContents(appeal, name).title}
+          </p>
+          {appeal.attributes.description && (
             <p className="vads-u-margin-y--0">
-              Status: {getStatusContents(appeal, name).title}
+              {appeal.attributes.issues.length === 1 ? 'Issue' : 'Issues'} on
```

Comment:
It makes me slightly nervous that we don't have any optional chaining in this file on these properties on the `appeal`.. shouldn't it?

Reply:
I remember you made a comment like this in the past because I'm having the same thought now. Which is: since this would be data we got from the API, if it _is_ malformed, I think we actually want to crash so we will have _any_ chance of discovering the fact that the API is no longer providing us data in the correct shape. But since is code was mostly copied from very old code, I'll take another look

Reply:
okay I'm gonna throw an TypeError at the top of this and the Claim component if it's passed a object that's missing `attributes`